kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KafkaRequestHandler.java

```diff
@@ -2466,11 +2491,44 @@ private void updateProducerStats(final TopicPartition topicPartition, final int
             case ALTER:
             case UNKNOWN:
             case ALL:
-            case ANY:
             default:
-                return FutureUtil.failedFuture(
-                        new IllegalStateException("AclOperation [" + operation.name() + "] is not supported."));
+                break;
+        }
+        if (isAuthorizedFuture == null) {
+            return FutureUtil.failedFuture(
+                    new IllegalStateException("AclOperation [" + operation.name() + "] is not supported."));
         }
         return isAuthorizedFuture;
     }
+
+    /**
+     * If we are using afkaEnableMultitenantMetadata we need to ensure
```

Comment:
Did you mean `kafkaEnableMultiTenantMetadata` ?

Reply:
@eolivelli Don't forget this. : )

Reply:
thanks